src/04_printing.py

```diff
@@ -11,7 +11,10 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
+print('x is %s, y is %s, z is "%s"' % (x, y, z))
```

Comment:
for these, industry standard would be to use %d for x, %.2f for y, and %s for z
how i think of them:
d = whole decimal number
.2f = float with a precision of 2 points beyond the decimal
s = string

this should still print everything the way you need, but there may *hint hint* come a time where you'll need to format your printed items with a specific type. ;)